tensorboard/webapp/widgets/markdown_renderer/markdown_renderer_component.ts

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component, Input, ViewEncapsulation} from '@angular/core';
+import {parse} from 'marked';
+
+@Component({
+  selector: 'markdown-renderer',
+  templateUrl: './markdown_renderer.ng.html',
+  styleUrls: ['./markdown_renderer.css'],
```

Comment:
change filenames to include `_component` suffix: i.e., `markdown_renderer_component.css`.

Reply:
done